src/cluster-map.tsx

```diff
@@ -33,18 +33,17 @@ export class ClusterMap extends React.PureComponent<
     const { style, region, priorityMarker } = this.props;
 
     return (
-      <GoogleMapView
+      <MapView
         {...utils.serializeProps(this.props)}
         ref={(ref) => (this.mapRef = ref)}
         style={style || styles.map}
         onMapReady={this.onMapReady}
         initialRegion={region}
         onRegionChangeComplete={this.onRegionChangeComplete}
-        provider={PROVIDER_GOOGLE}
```

Comment:
Could you add default provider for `google` because it'll be comfortable for users of previous version when they should add new prop.